Libraries/LibGUI/ModelIndex.cpp

```diff
@@ -46,4 +46,14 @@ const LogStream& operator<<(const LogStream& stream, const ModelIndex& value)
     return stream << String::format("ModelIndex(%d,%d)", value.row(), value.column());
 }
 
+} // namespace GUI
+
+void AK::Formatter<GUI::ModelIndex>::format(TypeErasedFormatParams& params, FormatBuilder& builder, const GUI::ModelIndex& value)
+{
+    Formatter<StringView> formatter { *this };
+
+    if (value.internal_data())
+        formatter.format(params, builder, String::formatted("ModelIndex({},{},{:p})", value.row(), value.column(), value.internal_data()));
+    else
+        formatter.format(params, builder, String::formatted("ModelIndex({},{})", value.row(), value.column()));
```

Comment:
This function would be more readable at the top of the file imo.

Reply:
What do you mean by that?

Reply:
It would be more clear that that function is not part of the GUI namespace.

Reply:
That's what the `// namespace GUI` comment is for.

Reply:
> That's what the // namespace GUI comment is for.

Doesn't really help since there is already a `}` at the end of the file and `// namespace xxx` is not required.